filter_directories.py

```diff
@@ -2,6 +2,12 @@
 
 import os
 import sys
+import lxml.etree as etree
+
+def is_tool(path):
+    xml = etree.parse(path)
+    return xml.getroot().tag == 'tool'
```

Comment:
Yes, ideally this should all be covered by some library so we can share this setup with multiple repositories. I'm fine if this is planemo. Are you joining the IUC meeting? This fits very much into my evil tools plans :)